### PR TITLE
Update istioctl upgrade docs

### PR DESCRIPTION
**Pull request by elfinhe**

### CODE REVIEW

File: content/en/docs/setup/upgrade/istioctl-upgrade/index.md [hunk 1]
```diff
@@ -1,27 +1,25 @@
...
-for all the options provided by the `istioctl experimental upgrade` command.
+See the [`istioctl` upgrade reference](/docs/reference/commands/istioctl/#istioctl-upgrade)
+for all the options provided by the `istioctl upgrade` command.
 
 ## Upgrade prerequisites
 
 Ensure you meet these requirements before starting the upgrade process:
 
-* Istio version 1.3.3 or higher is installed.
+* Istio version 1.4.4 or higher is installed.
```
Comment (sdake) [inline content/en/docs/setup/upgrade/istioctl-upgrade/index.md]: is this pre-req right? I don't recall upgrade being promoted from `istioctl experimental upgrade` in 1.4 series.

Comment (sdake) [reply content/en/docs/setup/upgrade/istioctl-upgrade/index.md]: I expanded the context and noticed this is are prereqs for the original installed version. Not sure how to clarify, but should be clarified if possible

Comment (elfinhe) [reply content/en/docs/setup/upgrade/istioctl-upgrade/index.md]: sorry, can you explain more, or should we add something like "Please upgrade your cluster to 1.4.4".

File: content/en/docs/setup/upgrade/istioctl-upgrade/index.md [hunk 2]
```diff
@@ -92,29 +90,23 @@ can be found in the `bin/` subdirectory of the downloaded package.
...
+For example, if you are downgrading from Istio 1.5 to 1.4.4, use `istioctl`
+version 1.4.4.
 
-### Downgrade to Istio 1.3.3 and lower versions steps
+## Downgrade to Istio 1.4.4 and lower versions steps
 
-The `istioctl experimental upgrade` command is not available in Istio 1.3.3 and lower.
-Therefore, downgrade must be performed using the
-`istioctl experimental manifest apply` command.
+You can use `istioctl experimental upgrade` to downgrade to 1.4 versions. Please
```
Comment (frankbu) [inline content/en/docs/setup/upgrade/istioctl-upgrade/index.md]:
```suggestion
You can use `istioctl upgrade` to downgrade to 1.4 versions. Please
```

**Pull request merged**